bin/git-summary

```diff
@@ -228,10 +224,9 @@ print_summary_by_line() {
     echo
     echo " project     : $project"
     echo " lines       : $(line_count "${paths[@]}")"
-    authors_file=$(git_extra_mktemp)
-    lines "${paths[@]}" | sort | uniq -c | sort -rn >${authors_file}
+    AUTHOR_LISTING=$(lines "${paths[@]}" | sort | uniq -c | sort -rn)
     echo " authors     : $(number_of_authors)"
-    format_authors <${authors_file}
+    format_authors
```

Comment:
Since AUTHOR_LISTING and format_authors are in the same function, could we pass AUTHOR_LISTING as an argument to format_authors?

Reply:
Yes, now both `format_authors` and `number_of_authors` take the listing as an argument.